### DISCUSSION TITLE
Schedule interval of a DAG

### DISCUSSION THREAD

**Discussion by panyu1512** [apache/airflow#36554] [Q&A]
It is possible in airflow set the schedule of a DAG the January 1st and then every Saturday at 5:00 until the end of March?

**Comment by broholens** [comment]
You can define your timetable. Check [timetable](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html) for more details.

**Reply by panyu1512** [reply]
Thank you!
